Makefile

```diff
@@ -26,10 +26,17 @@ $(notebooks): clean_notebooks lint mypy
 	$(CALLYSTO) $(LOCAL_ROOT_DIR)/$@ > $(LOCAL_ROOT_DIR)/$(@:.py=.ipynb)
 	docker exec -it $(CONTAINER) $(CONTAINER_REPO_DIR)/scripts/publish.sh $(CONTAINER_REPO_DIR)/$@ $(CONTAINER_REPO_DIR)/$(@:.py=.ipynb)
 
+cicd_notebooks:=$(notebooks:%=cicd_%)
```

Comment:
Makefile syntex is often difficult to understand at first glance.

Please consider adding a comment explaining what this line does.